README.md

```diff
@@ -13,16 +13,22 @@ layout:
     visible: true
 ---
 
-# Kontrol
-
 <div data-full-width="true">
 
 <figure><img src=".gitbook/assets/kontrol logo yellow.png" alt=""><figcaption></figcaption></figure>
 
 </div>
 
-[**Kontrol**](https://github.com/runtimeverification/kontrol) combines **KEVM** and [Foundry](https://book.getfoundry.sh/) to grant developers the ability to perform [formal verification](https://en.wikipedia.org/wiki/Formal\_verification) without learning a new language or tool. This is especially useful for those who are not verification engineers. Additionally, developers can leverage Foundry test suites they have already developed and use symbolic execution to increase the level of confidence.
+# Kontrol
+
+Struggling to kontrol your smart contracts behave as intended, even after thorough unit testing, fuzzing, and other validation procedures? Frustrated with finding bugs in your code but unable to prove their absence?
+
+[**Kontrol**](https://github.com/runtimeverification/kontrol) is designed for you!
+
 
-[**KEVM**](https://github.com/runtimeverification/evm-semantics) is a tool that enables formal verification of smart contracts on the [Ethereum](https://ethereum.org/en/) blockchain. It provides a mathematical foundation for specifying and implementing smart contracts. Developers can use **KEVM** to rigorously reason about the behavior of their smart contracts, ensuring correctness and reducing the likelihood of vulnerabilities in the contract code.
+- **Open Source**: Kontrol is open source under the [BSD-3 Clause License](https://github.com/runtimeverification/kontrol/blob/master/LICENSE). This gives you full Kontrol to use, modify, and redistribute it according to your needs.
+- **Easy to Use**: Take advantage of the languages and tests you’re already familiar with. Kontrol enables developers to write formal specifications as [Foundry](https://book.getfoundry.sh/) test suites in [Solidity](https://soliditylang.org). It allows developers to reuse their existing test suites for formal verification. Even if you're not a verification engineer, Kontrol lets you formally verify your contracts in minutes.
+- **Scalable**: Kontrol automatically performs theorem proving on your specifications against your contracts through composable symbolic execution. It allows you to manipulate the proof process and provide simplification rules to achieve any proof goal. This ensures you can verify your contracts at any scale.
```

Comment:
I think combining "theorem proving" and "composable symbolic execution" (I think we use "compositional" more commonly in the codebase/options/etc) might confuse people who have some familiarity with FV. Maybe we can instead something like this? 

```suggestion
- **Scalable**: Kontrol automatically performs verifies your contracts against the specification by using compositional symbolic execution. In addition to automated verification, Kontrol allows you to manipulate the proof process and provide simplification rules to achieve any proof goal, ensuring that you can verify your contracts at any scale.
```

Reply:
But doesn't this make our feature seem too vague? In my view, symbolic execution has its limitations, e.g., state space explosion or bounded depths. However, we do not. Even though we automatically prove the goals through symbolic execution, the theoretical foundation behind it is matching logic. 

1. The symbolic execution is indeed a proof generation process. 
2. The KCFG actually serves as a proof (or proof hint) in matching logic that can be verified using a proof checker. 

Therefore, the trust basis and actual implications are fundamentally different from traditional symbolic execution.

Reply:
```suggestion
- **Scalable**: Kontrol automatically verifies your contracts against your formal specifications via composable symbolic execution. Furthermore, Kontrol generates manipulable proofs of the verification process and enables users to provide simplification rules to achieve any proof goal. This guarantees that you can verify your contracts at any scale.
```